package.json

```diff
@@ -41,17 +41,18 @@
     "chai": "^4.1.2",
     "eslint": "^4.19.1",
     "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-plugin-import": "^2.9.0",
+    "eslint-plugin-import": "^2.11.0",
     "ganache-cli": "^6.1.0",
-    "mocha": "^5.0.5",
+    "mocha": "^5.1.1",
     "npm-watch": "^0.3.0",
-    "truffle": "^4.1.3"
+    "truffle": "^4.1.7"
   },
   "dependencies": {
-    "bignumber.js": "^6.0.0",
+    "bignumber.js": "^7.0.1",
     "ethereumjs-tx": "^1.3.4",
+    "hoek": "^5.0.3",
```

Comment:
In the `npm ls` tree, this was on the trunk, so I guess `dependencies` rather than `devDependenceis`?